lib/src/equatable_mixin.dart

```diff
@@ -15,6 +15,9 @@ mixin EquatableMixin {
   // ignore: avoid_returning_null
   bool? get stringify => null;
 
+  /// Cached hashCode.
+  int? _hashCode;
```

Comment:
By adding this mutable property, all `Equatable` instances are technically no longer immutable.

Reply:
The reason you don't see any warnings in the current PR is because we missed adding the `@immutable` annotation to `EquatableMixin`. I opened https://github.com/felangel/equatable/pull/154 to address this.

If you apply the changes from https://github.com/felangel/equatable/pull/154 to your branch you will see all the warnings reported.